README.md

```diff
@@ -13,14 +13,17 @@ A curated list of awesome things and projects built by Algerian developpers.
 ## Open Data
 - [Algerian Administrative Division](https://github.com/mohsenuss91/AlgerianAdministrativeDivision) - Wilayas - Daïras - baladiyahs
 - [Algerial Logos](https://github.com/mohsenuss91/AlgerianLogos) - Some Algerian official vectorized (.SVG) Logos
+- [algeria-cities](https://github.com/othmanus/algeria-cities) - A complete list of Algeria provinces and municipalities in csv, xlsx, sql, php and json formats.
 
 ## Applications
 - [Alfanous](https://github.com/Alfanous-team/alfanous) - Alfanous is an Arabic search engine API provides the simple and advanced search in Quran
+- [DzLibs](https://github.com/01walid/dzlibs) - A community-driven Algerian index of reusable assets and libraries.
```

Comment:
First, these rules are not written in stone and could be changed. But as mentioned on the README, for now, we're considering a project as active if it has at least one commit in the last two years. And I don't think a minor change just to appear as an active project is a problem, it shows that the author is still having some interest in the project (and want it to appear in this awesome list). I personally think it's a good comprise. At first I thought about a rule more like "commit in the last year" but I changed my mind because of have such a small community. The idea is to keep a decent list :).

BTW, I have several projects on GitHub that I consider good and I did not put them on this list, because I'm not working on them for now, so I can't really consider them as active.

I hope you'll understand ;)